.pylintdict

```diff
@@ -205,16 +207,22 @@ hartree
 hassidim
 hb
 hhl
+hijs
```

Comment:
What do you think about using a local `pylint:disable=invalid-name`? Some names you are adding here (such as `ia`) I think shouldn't be allowed globally.